cmd/scheduled-feed/main.go

```diff
@@ -131,14 +147,16 @@ func main() {
 	}
 }
 
-func cronRequest(port int) {
-	client := &http.Client{
-		Timeout: 10 * time.Second,
-	}
-	resp, err := client.Get(fmt.Sprintf("http://localhost:%v", port))
+func cronRequest(handler *FeedHandler) {
```

Comment:
This can be renamed now to reflect the changes in this PR as it no longer makes the http request.

Reply:
Agreed, changed to `cronPoll()`